docs/installing/adding_devices.md

```diff
@@ -26,13 +26,13 @@ The following instructions guide you through the process of installing the requi
 
 - x86_64
   - {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 22.x (jammy), Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 10 (buster), Debian 9 (stretch)
-  - {{site.data.keyword.rhel}} 8.1 - 8.5 (via Docker), 8.6 and 9.0 (via Podman 4.x).  See Notes.
+  - {{site.data.keyword.rhel}} 8.1 - 8.5 (via Docker), 8.6 - 8.7 and 9.0 - 9.1 (via Podman 4.x).  See Notes.
   - {{site.data.keyword.fedora}} Workstation 32, 36, 37
   - CentOS 8.1 - 8.5 (via Docker)
   - SuSE 15 SP2
 - ppc64le
   - {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
-  - {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 9.0
+  - {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 9.0, 9.1
```

Comment:
Do we need 8.7 listed here?

Reply:
Good eye for detail.  Yes, I've added `8.7` here.  I don't necessarily know how to test on ppc64le however.